### PR TITLE
Renaming modules, adding loader module, and adding basic angular web …

### PR DESCRIPTION
**Pull request by burzum619**
…app boiler plate

### CODE REVIEW

File: service/src/main/webapp/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# TdsAssessmentPackageLoader
+
```
Comment (trapnine) [inline service/src/main/webapp/README.md]: Rename?

File: service/src/main/webapp/e2e/app.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { AppPage } from './app.po';
+
+describe('tds-assessment-package-loader App', () => {
```
Comment (trapnine) [inline service/src/main/webapp/e2e/app.e2e-spec.ts]: rename?

File: service/src/main/webapp/package.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "tds-assessment-package-loader",
```
Comment (trapnine) [inline service/src/main/webapp/package.json]: rename

File: service/src/main/webapp/src/app/app.component.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import {Component} from "@angular/core";
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  title = 'TDS Assessment Package Loader';
```
Comment (trapnine) [inline service/src/main/webapp/src/app/app.component.ts]: rename

File: service/src/main/webapp/src/index.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>TdsAssessmentPackageLoader</title>
```
Comment (trapnine) [inline service/src/main/webapp/src/index.html]: rename

File: service/src/main/webapp/tslint.json [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    ],
+    "prefer-const": true,
+    "quotemark": [
+      true,
+      "single"
+    ],
+    "radix": true,
+    "semicolon": [
+      true,
+      "always"
```
Comment (mtrupkin) [inline service/src/main/webapp/tslint.json]: prefer "never"

Comment (millerts) [reply service/src/main/webapp/tslint.json]: Are you requesting that JavaScript no have semicolons?

Comment (trapnine) [reply service/src/main/webapp/tslint.json]: this controls lint complaining about semicolons on EOL? is it an error or warning? I think we should use the semicolons, in general, as a style.

**Pull request merged**